docker/luigi_docker.cfg

```diff
@@ -0,0 +1,35 @@
+[core]
+log_level = INFO
+logging_conf_file = /app/docker/luigi_logging_docker.cfg
+
+[scheduler]
+state_path = /var/lib/luigi/luigi.state
+
+[email]
+format = html
+method = ses
+prefix = [PRODUCTION SYSTEMS]
+# update these to a central email address and then hide this file from git
+receiver = 
```

Comment:
how will `sender` and `receiver` be set?

Reply:
This is revised. It will pull in the host configurations and therefore set these fields from there. I have removed the docker specific `luigi.cfg`.

There is still a docker specific logging config file, which is passed in as a flag when the task is called.